fixtures/shehds/shedhds-led-beam-wash-19x15-rugby-zoom.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "shedhds led beam wash 19x15 rugby zoom",
```

Comment:
I can't find any Shehds fixture with "rugby" in the name and I don't know where that comes from or what it means.